Xamarin.Forms.Platform.UAP/Properties/AssemblyInfo.cs

```diff
@@ -7,7 +7,9 @@
 // Views
 
 [assembly: ExportRenderer(typeof(Layout), typeof(LayoutRenderer))]
+#pragma warning disable CS0618 // Type or member is obsolete
 [assembly: ExportRenderer(typeof(BoxView), typeof(BoxViewRenderer))]
```

Comment:
@samhouts  Should I remove disabling that warning and change the default renderer to the new one here also?